src/repositories/powerBoostingRepository.ts

```diff
@@ -64,8 +64,8 @@ export const findUsersWhoBoostedProject = async (
 };
 
 export const findPowerBoostings = async (params: {
-  take: number;
-  skip: number;
+  take?: number;
+  skip?: number;
```

Comment:
Are you fetching them all in one go? wouldn't this list be way too huge in the future? maybe you should set a default take.